Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM ubuntu:latest
 MAINTAINER William Budington "bill@eff.org"
 
+RUN echo "deb http://deb.torproject.org/torproject.org $(lsb_release -sc) main" > /etc/apt/sources.list.d/tor.list && \
+  echo "deb-src http://deb.torproject.org/torproject.org $(lsb_release -sc) main" >> /etc/apt/sources.list.d/tor.list
```

Comment:
`lsb_release` is not found.  Please test this locally first by running

```bash
sudo docker build -t electronicfrontierfoundation/https-everywhere-docker-base .
```

within your local repository.